src/components/popover/popover.tsx

```diff
@@ -77,9 +77,18 @@ export class Popover implements FloatingUIComponent, OpenCloseComponent {
    */
   @Prop({ reflect: true }) dismissible = false;
 
+  handleDismissible(value: boolean): void {
```

Comment:
does this need a watcher?

Reply:
This needs the watcher to update `closable` value .

Reply:
Is it missing?

Reply:
Ahh i see it now, missed the decorator.

Reply:
Will add an e2e test to make sure both props are in sync.